conda/environments/builddocs_py37.yml

```diff
@@ -12,8 +12,9 @@ dependencies:
 - pandas>=0.23.4
 - pyarrow=0.12.0
 - boost
-- cffi>=1.10.0   
-- distributed>=1.23.0
+- cffi>=1.10.0
```

Comment:
Is cffi still needed? It is not in any of the conda recipe definitions for packages.

Reply:
These changes are not part of this PR, cf Dante's comment